.idea/vcs.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
This file should not exist in the repository because it is internal to your IDE. You should add the whole directory to the `.gitignore`.

Reply:
Even better add it to your global gitignore so that it doesn't clutter repo ones: https://help.github.com/articles/ignoring-files/